src/net.cpp

```diff
@@ -360,6 +360,15 @@ CNode* FindNode(const CNetAddr& ip)
     return NULL;
 }
 
+CNode* FindNode(const CSubNet& subNet)
+{
+    LOCK(cs_vNodes);
+    BOOST_FOREACH(CNode* pnode, vNodes)
```

Comment:
I know this is from upstream, but we shouldn't ever add `BOOST_FOREACH` to the codebase and should replace it when we can with built in `for` range search in c++11. `for (CNode* pnode : vNodes)`

Reply:
yep, was planning on replacing the `BOOST_FOREACH`, just wanted to get this up ASAP for functionality testing.